Command/PasswordResetRequestHandler.php

```diff
@@ -76,6 +85,9 @@ private function sendPasswordResetToken(PasswordResetInterface $user)
      */
     private function getPasswordResetMessage(PasswordResetInterface $user)
     {
-        return str_replace('%token%', $user->getPasswordResetToken(), $this->bodyText);
+        $url = $this->router->generate('multi_user_reset_password');
+        $token = '?token='.$user->getPasswordResetToken();
```

Comment:
dit zal ook properder zijn als ge je token in je url verwerkt. Niet meer handmatig url's opbouwen :) (ge kunt trouwens ook GET parameters meegeven aan `router->generate`